bw2h5.py

```diff
@@ -70,30 +80,48 @@ def parse_args():
     parser = argparse.ArgumentParser(
         description='Data processing for genome-wide denoising models.')
     parser.add_argument('--noisybw', type=str,
-                        help='Path to noisy bigwig file', required=True)
+                        help='Path to bigwig file containing noisy \
+                        (low coverage/low quality) ATAC-seq signal',
+                        required=True)
     parser.add_argument('--layersbw', type=str,
                         help='Paths to bigWig files containing \
                             additional layers. If single file,  \
                             use format: "name:file". \
                             If there are multiple files, use format: \
                             "[name1:file1, name2:file2,...]"')
     parser.add_argument('--intervals', type=str,
-                        help='Path to interval file', required=True)
+                        help='Path to BED file containing genomic intervals. \
+                        ATAC-seq data within these intervals will be read \
+                        from the bigWig files. See get_intervals.py for \
+                        help generating such a BED file.',
+                        required=True)
     parser.add_argument('--batch_size', type=int,
-                        help='batch size', required=True)
-    parser.add_argument('--pad', type=int, help='padding around interval')
+                        help='batch size; number of intervals to read from bigWig \
+                        files at a time. Unrelated to training/inference \
+                        batch size.', default=1000)
+    parser.add_argument('--pad', type=int, help='Number of additional bases to \
+                        add as padding on either side of interval. Use the \
+                        same value for training, validation and test files.')
+    parser.add_argument('--out_home', type=str,
```

Comment:
So do I actually. I used `out_home` because that's what we've been using in `main.py` from the beginning and I wanted it to be uniform.

Reply:
There are other inconsistencies - `--sizes` is used for the chromosome sizes file in preprocessing scripts and `--sizes_file` in `main.py`

Reply:
Hmm I see. Let's keep it as `out_dir` since it's more descriptive. may be too late to change `main.py`, but at least the other scripts will have better naming :)

Reply:
done